api/src/main/java/com/xwiki/azureoauth/internal/EntraIDObjectUpdateListener.java

```diff
@@ -150,4 +172,38 @@ private WikiReference getCurrentWikiReference()
     {
         return new WikiReference(this.wikiManager.getCurrentWikiId());
     }
+
+    private void resetAuthService(XWiki xwiki) throws ComponentLookupException
```

Comment:
I'd add a comment to explain why we need to reset the auth service.
Also I'd mention what part needs to be removed once we upgrade to a XWiki version that doesn;t need the backport extension